imagedephi/redact/redact.py

```diff
@@ -37,6 +38,19 @@ def iter_image_files(directory: Path) -> Generator[Path, None, None]:
             yield child
 
 
+def create_redact_dir(base_output_dir: Path) -> Path:
+    """Given a directory return a timestamped directory created in the base directory."""
+    time_stamp = datetime.datetime.now().strftime("%Y-%m-%d_%H-%M-%S")
+    redact_dir = base_output_dir / f"Redacted_{time_stamp}"
+    try:
+        redact_dir.mkdir(parents=True)
+        click.echo(f"Created redaction folder: {redact_dir}")
```

Comment:
Since this `click.echo` isn't expected to fail, it shouldn't go inside the `try`.

Consider putting it inside an `else` if you want to keep it associated with the overall `try` block.